README.md

```diff
@@ -0,0 +1,40 @@
+# OLSR daemon
+
+# How to Install
+
+## Installing from source
+
+Get your code from the OLSR GitHub:
+(see https://github.com/OLSR/olsrd)
+
+ * ``git clone https://github.com/OLSR/olsrd.git``
+
+### Pre-requisites
+
+Download and install the neccessary build requirements
+
+For Debian you will mostly need the following ones:
+  * build-essentials: ``sudo apt-get install build-essential dpkg-dev``
+
+## Compiling
+
+  * ``cd olsrd``
+  * ``make``
+  * (plugins) ``cd lib; for i in $(ls); do cd "$i"; make; cd ..; done``
+
+## Configuring OLSRd
+
+## Starting OLSRd
+
+Assuming your interfaces you want olsrd to listen on are ``eth0, wlan0 and lo`` you could start it like this:
+
+  * ``sudo ./olsrd_static -i eth0 wlan0 lo -d 0 -f olsrd.conf``
```

Comment:
did you check this?
I think it should be "... -i eth0 -i wlan0 -i lo ...."

Reply:
The -f is not needed; it will look in /etc/olsrd/olsrd.conf

Reply:
see e.g. https://www.apt-browse.org/browse/debian/wheezy/main/i386/olsrd/0.6.2-2.1/file/etc/olsrd/olsrd.conf
using the debian package the interfaces are defined this way and this worked for me some time ago.
i didnt try multiple -i options though.

Reply:
removed -f option.